### ISSUE TITLE
Can't create ticket - GLPI 10.0.12

### ISSUE THREAD

**Issue by tech62** [glpi-project/glpi#16581]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.6 to 10.0.12

### Bug description

As super-admin , i'm creating a new ticket, filling all fields , clicking on Create and nothing happenned.

### Relevant log output

```shell
SQL: INSERT INTO `glpi_tickets` (`name`, `content`, `entities_id`, `date`, `type`, `itilcategories_id`, `status`, `requesttypes_id`, `urgency`, `priority`, `locations_id`, `actiontime`, `time_to_own`, `slas_id_tto`, `slas_id_ttr`, `olas_id_tto`, `olas_id_ttr`, `impact`, `users_id_lastupdater`, `users_id_recipient`, `global_validation`, `slalevels_id_ttr`, `sla_waiting_duration`, `time_to_resolve`, `date_creation`, `date_mod`) VALUES ('test', '&#60;p&#62;test&#60;/p&#62;', '0', '2024-02-19 09:49:11', '1', '70', '1', '1', '3', '3', '0', '0', '2024-02-19 14:49:11', '16', '17', '0', '0', '3', '2096', '2096', '1', '0', '',
'2024-02-19 11:49:11', '2024-02-19 09:49:22', '2024-02-19 09:49:22')

  Error: Incorrect integer value: '' for column `glpi`.`glpi_tickets`.`sla_waiting_duration` at row 1
  Backtrace :
  src/DBmysql.php:1380                               DBmysql->doQuery()
  src/CommonDBTM.php:729                             DBmysql->insert()
  src/CommonDBTM.php:1342                            CommonDBTM->addToDB()
  front/ticket.form.php:80                           CommonDBTM->add()
  public/index.php:82                                require()
```
```


### Page URL

_No response_

### Steps To reproduce

As super-admin , i'm creating a new ticket, filling all fields , clicking on Create and nothing happenned.

### Your GLPI setup information

<details><summary>Informations sur le système, l'installation et la configuration</summary><pre>GLPI 10.0.12 ( => /var/www/glpi)
Installation mode: TARBALL
Current language:fr_FR

</pre></details><details><summary>Server</summary><pre> 
Operating system: Linux SRV-SUPPORT 6.1.0-18-amd64 #​1 SMP PREEMPT_DYNAMIC Debian 6.1.76-1 (2024-02-01) x86_64
PHP 8.2.7 fpm-fcgi (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, bcmath, bz2, calendar, cgi-fcgi, ctype,
	curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, igbinary, imap, intl, json, ldap, libxml, mbstring,
	mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, random, readline, redis, session, shmop, snmp, soap, sockets, sodium,
	standard, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="300" memory_limit="2G" post_max_size="1G" safe_mode="" session.save_handler="files"
	upload_max_filesize="1G" disable_functions="" 
Software: Apache ()
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
Server Software: Debian 12
	Server Version: 10.5.21-MariaDB-0+deb11u1
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: glpi@mysql/glpi
	Host info: mysql via TCP/IP
	
PHP version (8.2.7) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.
curl extension is installed.
gd extension is installed.
intl extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.5.21) is supported.
No files from previous GLPI version detected.
The log file has been created successfully.
Write access to /var/lib/glpi/prod/files/_cache has been validated.
Write access to /var/lib/glpi/prod/files/_cron has been validated.
Write access to /var/lib/glpi/prod/files has been validated.
Write access to /var/lib/glpi/prod/files/_dumps has been validated.
Write access to /var/lib/glpi/prod/files/_graphs has been validated.
Write access to /var/lib/glpi/prod/files/_lock has been validated.
Write access to /var/lib/glpi/prod/files/_pictures has been validated.
Write access to /var/lib/glpi/prod/files/_plugins has been validated.
Write access to /var/lib/glpi/prod/files/_rss has been validated.
Write access to /var/lib/glpi/prod/files/_sessions has been validated.
Write access to /var/lib/glpi/prod/files/_tmp has been validated.
Write access to /var/lib/glpi/prod/files/_uploads has been validated.

Web server root directory configuration seems safe.
Sessions configuration is secured.
OS and PHP are relying on 64 bits integers.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/glpi/marketplace has been validated.
Timezones seems loaded in database.

</pre></details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: "/var/www/glpi"
GLPI_CONFIG_DIR: "/etc/glpi/prod/"
GLPI_VAR_DIR: "/var/lib/glpi/prod/files"
GLPI_LOG_DIR: "/var/log/glpi/prod"
GLPI_MARKETPLACE_DIR: "/var/www/glpi/marketplace"
GLPI_USE_CSRF_CHECK: "1"
GLPI_CSRF_EXPIRES: "7200"
GLPI_CSRF_MAX_TOKENS: "100"
GLPI_USE_IDOR_CHECK: "1"
GLPI_IDOR_EXPIRES: "7200"
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: false
GLPI_SERVERSIDE_URL_ALLOWLIST: ["/^(https?|feed):\\/\\/[^@:]+(\\/.*)?$/"]
GLPI_TELEMETRY_URI: "https://telemetry.glpi-project.org"
GLPI_INSTALL_MODE: "TARBALL"
GLPI_NETWORK_MAIL: "glpi@teclib.com"
GLPI_NETWORK_SERVICES: "https://services.glpi-network.com"
GLPI_MARKETPLACE_ALLOW_OVERRIDE: true
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: true
GLPI_USER_AGENT_EXTRA_COMMENTS: ""
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: "1"
GLPI_AJAX_DASHBOARD: "1"
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: "0"
GLPI_CENTRAL_WARNINGS: "1"
GLPI_TEXT_MAXSIZE: "4000"
GLPI_DOC_DIR: "/var/lib/glpi/prod/files"
GLPI_CACHE_DIR: "/var/lib/glpi/prod/files/_cache"
GLPI_CRON_DIR: "/var/lib/glpi/prod/files/_cron"
GLPI_DUMP_DIR: "/var/lib/glpi/prod/files/_dumps"
GLPI_GRAPH_DIR: "/var/lib/glpi/prod/files/_graphs"
GLPI_LOCAL_I18N_DIR: "/var/lib/glpi/prod/files/_locales"
GLPI_LOCK_DIR: "/var/lib/glpi/prod/files/_lock"
GLPI_PICTURE_DIR: "/var/lib/glpi/prod/files/_pictures"
GLPI_PLUGIN_DOC_DIR: "/var/lib/glpi/prod/files/_plugins"
GLPI_RSS_DIR: "/var/lib/glpi/prod/files/_rss"
GLPI_SESSION_DIR: "/var/lib/glpi/prod/files/_sessions"
GLPI_TMP_DIR: "/var/lib/glpi/prod/files/_tmp"
GLPI_UPLOAD_DIR: "/var/lib/glpi/prod/files/_uploads"
GLPI_INVENTORY_DIR: "/var/lib/glpi/prod/files/_inventories"
GLPI_NETWORK_REGISTRATION_API_URL: "https://services.glpi-network.com/api/registration/"
GLPI_MARKETPLACE_PLUGINS_API_URI: "https://services.glpi-network.com/api/marketplace/"
GLPI_I18N_DIR: "/var/www/glpi/locales"
GLPI_VERSION: "10.0.12"
GLPI_SCHEMA_VERSION: "10.0.12@b902ef5f5da967c1a9d0826ee8a7dce07d57cf3d"
GLPI_MARKETPLACE_PRERELEASES: false
GLPI_MIN_PHP: "7.4.0"
GLPI_MAX_PHP: "8.4.0"
GLPI_YEAR: "2024"

</pre></details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.14 in (/var/www/glpi/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.8.0 in (/var/www/glpi/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/glpi/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.6.2 in (/var/www/glpi/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/glpi/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/glpi/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/glpi/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/glpi/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/glpi/vendor/sabre/http/lib)
sabre/uri in (/var/www/glpi/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/glpi/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/glpi/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/glpi/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/glpi/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/glpi/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/glpi/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/glpi/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/glpi/vendor/symfony/console)
scssphp/scssphp in (/var/www/glpi/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/glpi/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/glpi/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/glpi/vendor/rlanvin/php-rrule/src)
ramsey/uuid in (/var/www/glpi/vendor/ramsey/uuid/src)
psr/log in (/var/www/glpi/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/glpi/vendor/psr/simple-cache/src)
psr/cache in (/var/www/glpi/vendor/psr/cache/src)
league/csv in (/var/www/glpi/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/glpi/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/glpi/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/glpi/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/glpi/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/glpi/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/glpi/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/glpi/vendor/symfony/cache)
html2text/html2text in (/var/www/glpi/vendor/html2text/html2text/src)
symfony/css-selector in (/var/www/glpi/vendor/symfony/css-selector)
symfony/dom-crawler in (/var/www/glpi/vendor/symfony/dom-crawler)
twig/twig in (/var/www/glpi/vendor/twig/twig/src)
twig/string-extra in (/var/www/glpi/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 not found
symfony/polyfill-php81 not found
symfony/polyfill-php82 in (/var/www/glpi/vendor/symfony/polyfill-php82)
league/oauth2-client in (/var/www/glpi/vendor/league/oauth2-client/src/Provider)
league/oauth2-google in (/var/www/glpi/vendor/league/oauth2-google/src/Provider)
thenetworg/oauth2-azure in (/var/www/glpi/vendor/thenetworg/oauth2-azure/src/Provider)

</pre></details><details><summary>Plugins list</summary><pre> 
	webapplications      Name: Applications Web               Version: 4.0.1      State: Enabled                                 
		Install Method: Marketplace
	badges               Name: Badges                         Version: 3.0.0      State: Enabled                                 
		Install Method: Marketplace
	screenshot           Name: Capture d'écran                Version: 2.0.3      State: Enabled                                 
		Install Method: Marketplace
	positions            Name: Cartographie                   Version: 6.0.3      State: Enabled                                 
		Install Method: Marketplace
	behaviors            Name: Comportements                  Version: 2.7.2      State: Enabled                                 
		Install Method: Manual
	accounts             Name: Comptes                        Version: 3.0.4      State: Enabled                                 
		Install Method: Marketplace
	mydashboard          Name: Dashboard                      Version: 2.1.5      State: Enabled                                 
		Install Method: Marketplace
	datainjection        Name: Data Injection                 Version: 2.13.4     State: Enabled                                 
		Install Method: Marketplace
	consumables          Name: Demande de consommables        Version: 2.0.1      State: Enabled                                 
		Install Method: Marketplace
	archimap             Name: Diagrammes                     Version: 3.3.4      State: Enabled                                 
		Install Method: Marketplace
	formcreator          Name: Form Creator                   Version: 2.13.8     State: Enabled                                 
		Install Method: Marketplace
	gantt                Name: gantt                          Version: 1.0.4      State: Enabled                                 
		Install Method: Marketplace
	geninventorynumber   Name: Génération de numéros d'invent Version: 2.8.3      State: Enabled                                
		 Install Method: Marketplace
	order                Name: Gestion des commandes          Version: 2.10.4     State: Enabled                                 
		Install Method: Marketplace
	glpiinventory        Name: GLPI Inventory                 Version: 1.3.4      State: Enabled                                 
		Install Method: Marketplace
	pdf                  Name: Impression pdf                 Version: 3.0.0      State: Installed / not activated               
		Install Method: Marketplace
	mreporting           Name: Plus de rapports               Version: 1.8.6      State: Enabled                                 
		Install Method: Marketplace
	webresources         Name: Web Resources                  Version: 2.0.3      State: Enabled                                 
		Install Method: Marketplace
	webhook              Name: Webhooks                       Version: 1.0.18     State: Enabled                                 
		Install Method: Marketplace

</pre></details>

### Anything else?

_No response_

**Comment by trasher** [comment]
This is already fixed; search in existing issues;

**Comment by tech62** [comment]
I've updated glpi to the latest version, how to fix ?

**Comment by cedric-anne** [comment]
Fixed by #16496.
